src/wallet/wallet.cpp

```diff
@@ -2695,7 +2695,7 @@ bool CWallet::CreateTransaction(const vector<CRecipient>& vecSend, CWalletTx& wt
 
                 if (nChange > 0)
                 {
-                    // Fill a vout to ourself
+                    // Fill a vout to ourselves
```

Comment:
This is largely a matter of taste, but I prefer "ourself", since the referent is grammatically singular.